### PR TITLE
A few updates, mostly on IBM stuff.

### PR DESCRIPTION
**Pull request by dmbaturin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -456,6 +464,8 @@ Needless to say, engaging the brake while the drum was spinning would be _bad id
 
 ####  008C Ever physically destroy equipment from software?
 
+It may sound funny now, but back in the days hardware wasn't as smart and it was possible to physically damage certain CRT displays by switching them into unusual modes, for example.
```
Comment (Screwtapello) [inline README.md:487]: See also question 00A5.

File: README.md [hunk 2]
```diff
@@ -1118,6 +1158,8 @@ the disk itself while you were cutting out the new notch...
 
 ####  01AC Do you use the "P convention"?
 
+Common Lisp and some other languages use the "p" suffix for predicates, e.g. `(numberp 42)`. Newer lisps like Scheme and Clojure usually go with more intuitive "?" instead, like `(string? "foo")`.
```
Comment (Screwtapello) [inline README.md:1193]: See also the Jargon File: http://catb.org/jargon/html/p-convention.html

File: README.md (hunk 1 above)
Comment (dmbaturin) [reply README.md:487]: Added a reference. It would be nice to properly cross-reference the whole thing...

File: README.md (hunk 2 above)
Comment (dmbaturin) [reply README.md:1193]: Added.

**Comment by hwayne** [comment]
Thank you!

**Pull request merged**